src/backend/expungeservice/models/charge_types/contempt_of_court.py

```diff
@@ -0,0 +1,21 @@
+from dataclasses import dataclass
+
+from expungeservice.models.charge import Charge
+from expungeservice.models.expungement_result import TypeEligibility, EligibilityStatus
+
+
+
+@dataclass(eq=False)
+class ContemptOfCourt(Charge):
+    """ This is a civil offense that is always ineligible regardless of level / conviction status"""
```

Comment:
We don't use class docstrings anywhere else, so I would remove this but use it as the `expungement_rules`,

Reply:
Done with latest commit